tests/post_training/test_templates/test_smooth_quant.py

```diff
@@ -92,10 +114,8 @@ def get_quantization_algorithm():
             (
                 LinearMultiShapeModel,
                 {
-                    "/Reshape_0_0/nncf_smooth_quant": [[[[1.0594617, 1.1019668, 1.2208323, 1.1003988]]]],
-                    "/Split_1_0/nncf_smooth_quant": [[[[1.1276343, 0.7605822]]]],
-                    "/Split_0_0/nncf_smooth_quant": [[[[0.32575992, 0.33121374]]]],
-                    "/Reshape_1_0_0/nncf_smooth_quant": [
+                    ("MatMul1", "MatMul2"): [[[[1.0594617, 1.1019668, 1.2208323, 1.1003988]]]],
```

Comment:
It's not obvious why we need to create a mapping from the regular layers to the new inserted by `SmoothQuant`.

Reply:
I did this because the Torch and OV models have different names. I made a map from an "node mark" ("MatMul1", "Linear1") to actual `NNCFNodeName` to keep references in common part of the test